src/java/org/apache/cassandra/metrics/CassandraMetricsRegistry.java

```diff
@@ -186,6 +186,8 @@ else if (metric instanceof Timer)
             mbean = new JmxTimer((Timer) metric, name, TimeUnit.SECONDS, TimeUnit.MICROSECONDS);
         else if (metric instanceof Metered)
             mbean = new JmxMeter((Metered) metric, name, TimeUnit.SECONDS);
+        else if (metric instanceof IntegerSetting)
```

Comment:
Codahale metrics are typically a read only thing, I think we can certainly go this route but then we have a 1 value in all of Cassandra set this way and I would prefer to see us move to Virtual Tables than to work to refactor all our JMX MBeans. The JMXEnabledThreadPoolExecutor also already has a `setMaximumThreads` so this will be confusing.